### ISSUE TITLE
../fast-export/hg-fast-export.sh: 156: python: not found

### ISSUE THREAD

**Issue by jane-han-obj** [frej/fast-export#228]
I have a error when I migrate the mercurial to git. I block on this step -- ../fast-export/hg-fast-export.sh -r ../demoapp

Windows 10 64 bit
Python 2.7.6 installed and in the path
Mercurial 4.8 install, and in the path
Git 2.26.2.windows.1 installed, and in the path
Running the following command in git-bash:

../fast-export/hg-fast-export.sh -r ../demoapp/ --fe cp1251

Looking at ../fast-export/hg-fast-export.sh: 156: python: not found, so I am not sure how to fix it. If anyone can help to figure out what's reason about this issue?
Any help appreciated.

**Comment by frej** [comment]
Current master does not have anything that could print that error message on line 156. Are you sure you are running the latest version of fast-export?

**Comment by frej** [comment]
I remembered #160 and #143 which could be duplicates of this one.

**Comment by jane-han-obj** [comment]
Hi @frej ,
Thanks you so much for your quick response. This issue has been fixed. It caused by I install the Python on the windows system and set path also on windows. I also downloaded the fast export code version -- tags/v180317.  I used the git bash to export the code, the hg-fast-export.sh cannot find the python. I used the Linux VM to run it and download the latest fast export on my local. It works now. I can export the mercurial repository code to git. Appreciate for your help.

Regards,
Jane
